garden/index.yaml

```diff
@@ -41,9 +41,6 @@ pages:
       - name: troubleshooting
         title: Troubleshooting
         file: troubleshooting.md
-      - name: ign_docker_env
-        title: Dockerized Dev Env
-        file: ign_docker_env.md
```

Comment:
This shouldn't be deleted right? I see the file is still there, it's just now been renamed to `gz_docker_env.md`?

Reply:
Ah, meant to delete the file, I was told that the ign_docker was not being updated/maintained for gz-garden, hence the removal.